packages/cli/src/commands/init/init.ts

```diff
@@ -213,6 +214,15 @@ export default (async function initialize(
   [projectName]: Array<string>,
   options: Options,
 ) {
+  if (!projectName) {
+    const {projName} = await prompts({
+      type: 'text',
+      name: 'projName',
+      message: 'What is the project name?',
```

Comment:
Would that wording make sense?
```suggestion
      message: 'How would you like to name the app?',
```
Also, how about adding a `hint` guiding the user on what is about to happen? Something like:

> New app will be initialized in the directory of the same name. Android and iOS projects will use this name for publishing setup

If this wording works, we can also reflect it in the command description

Reply:
> Also, how about adding a hint guiding the user on what is about to happen? Something like:

Fun fact: when using `type: 'text'` prompt, we cannot show the hint 🙈

> If this wording works, we can also reflect it in the command description

Done ✅